Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/types.go

```diff
@@ -2166,11 +2166,15 @@ type SecurityContextConstraints struct {
 	ObjectMeta `json:"metadata,omitempty"`
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
-	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer,omitempty" description:"allow containers to run as privileged"`
+	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer" description:"allow containers to run as privileged"`
```

Comment:
we should not have `json:...` annotations in pkg/api/types.go, only in versioned apis.